src/components/Form.jsx

```diff
@@ -1,17 +1,30 @@
-const Form = () => (
-  <div className="form-container">
-    <h2>Add a new book</h2>
-    <form>
-      <input type="text" name="name" placeholder="Book title" />
-      <input type="text" name="author" placeholder="Author" />
-      <select className="selection">
-        <option>Category</option>
-      </select>
-      <button className="add-btn" type="submit">
-        Add
-      </button>
-    </form>
-  </div>
-);
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const Form = () => {
+  const dispatch = useDispatch();
+  const [bookTitle, setBookTitle] = useState();
+  const [bookAuthor, setBookAuthor] = useState();
```

Comment:
- [ ] Try making your form input a controlled state, I think you left your initial state empty and it is throwing an error in the console. kindly change to `const [bookTitle, setBookTitle] = useState(' ');`. 
this is how it looks currently
    
![image](https://user-images.githubusercontent.com/66097371/161141430-51a761ef-7ce9-4da7-b2b4-5205493730f1.png)